.yamllint

```diff
@@ -11,3 +11,5 @@ rules:
   # NOTE(retr0h): Templates no longer fail this lint rule.
   #               Uncomment if running old Molecule templates.
   # truthy: disable
+ignore: |
+  .github/
```

Comment:
How about we don't ignore this directory. It has some yaml files that we'd want to check for correctness.